e2e/passcode/check-passcode-requirements.yaml

```diff
@@ -0,0 +1,40 @@
+appId: com.comapeo
+---
+- stopApp
+- launchApp:
+    appId: com.comapeo
+- assertVisible:
+    label: 'Passcode entry screen appears when user closes Mapeo app completely, and re-opens CoMapeo app'
+    text: 'Enter your passcode'
+- runFlow: './subflows/check-number-keyboard.yaml'
+- pressKey: Home
+- swipe:
+    direction: UP
+- tapOn: 'Clock'
+- pressKey: Home
+- swipe:
+    direction: UP
```

Comment:
pretty cool that you can simulate this! the `tapOn: 'Clock'` is probably not necessary but not adamant about removing

Reply:
Ok. It was in the script that another app was opened, so that is why I did it...